.github/actions/checkov-scans-v1/action.yml

```
@@ -0,0 +1,31 @@
+name: checkov-scans
+description: Checkov scans cloud infrastructure configurations to find misconfigurations before they're deployed and sends a result to Prisma Cloud.
+
+inputs:
+  prisma-access-key:
+    description: Prisma Cloud access key
+    required: true
+  prisma-secret-key:
+    description: Prisma Cloud secret key
+    required: true
+  prisma-api-url:
+    description: Prisma Cloud API URL
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so follow-up steps can access it
+    - uses: actions/checkout@v4
+    # The checkov scan
+    - name: Run Prisma Cloud
+      id: prisma-cloud
+      uses: bridgecrewio/checkov-action@8b268e1e4d7e52a78ba9d86e2489fb09b8a340c3 # tag=v12.2690.0
```

The pinned SHA is not quite accurate you'd want [99bb2caf247dfd9f03cf984373bc6043d4e32ebf](https://github.com/bridgecrewio/checkov-action/commit/99bb2caf247dfd9f03cf984373bc6043d4e32ebf) which is the latest on the [v12](https://github.com/bridgecrewio/checkov-action/releases/tag/v12.1347.0) tag
```suggestion
      uses: bridgecrewio/checkov-action@99bb2caf247dfd9f03cf984373bc6043d4e32ebf # tag=v12
```

---

Ignore me! This is the latest I missed that they released [yesterday](https://github.com/bridgecrewio/checkov-action/releases/tag/v12.2690.0)